harness/determined/deploy/gke/cli.py

```diff
@@ -0,0 +1,596 @@
+                            type=int,
+                            default=defaults.GPUS_PER_NODE,
+                            required=False,
+                            help="number of GPUs per node",
+                        ),
+                        Arg(
+                            "--helm-dir",
+                            type=str,
+                            default="helm/charts/determined",
+                            required=False,
+                            help="directory containing Helm Chart, values.yaml and templates.",
+                        ),
+                        Arg(
+                            "--det-version",
+                            type=str,
+                            default=None,
+                            help=argparse.SUPPRESS,
+                        ),
+                        Arg(
+                            "--no-manage-bucket",
+                            required=False,
+                            help="flag that indicates GCS checkpointing bucket already exists",
+                            action="store_true",
+                        ),
+                        Arg(
+                            "--zone",
+                            type=str,
+                            default=defaults.ZONE,
+                            help="zone to create cluster in",
+                        ),
+                        Arg(
+                            "--master-machine-type",
+                            type=str,
+                            default=defaults.MASTER_MACHINE_TYPE,
+                            help="machine type to use for master node group",
+                        ),
+                        Arg(
+                            "--agent-machine-type",
+                            "--machine-type",
+                            type=str,
+                            default=defaults.AGENT_MACHINE_TYPE,
+                            help="machine type to use for agent node group",
+                        ),
+                        Arg(
+                            "--max-gpu-nodes",
+                            "--max-nodes",
+                            type=int,
+                            default=defaults.MAX_GPU_NODES,
+                            help="maximum number of nodes for the GPU node group",
+                        ),
+                        Arg(
+                            "--max-cpu-nodes",
+                            type=int,
+                            default=defaults.MAX_CPU_NODES,
+                            help="maximum number of nodes for the CPU node group",
+                        ),
+                        Arg(
+                            "--cpu-node-pool-name",
+                            type=str,
+                            default=None,
+                            help="a unique name for the GPU node pool",
+                        ),
+                        Arg(
+                            "--multi-node-pool",
```

Comment:
--multi-node-pool sounds like it's referring to a pool with >1 nodes, not multiple node pools. Would suggest --multiple-node-pools. I suppose the reason to not just do both like we do for other platforms is because auto-scaling can't work with our recommended scheduler?